packages/core/drizzle/0087_legacy_evaluations_part_2.sql

```diff
@@ -0,0 +1,11 @@
+--- Migrate configuration column from "evaluations" to "llm_as_judge_evaluation_metadatas"
+UPDATE "latitude"."llm_as_judge_evaluation_metadatas"
+SET "configuration" = "latitude"."evaluations"."configuration"
+FROM "latitude"."evaluations"
+WHERE "latitude"."evaluations"."metadata_id" = "latitude"."llm_as_judge_evaluation_metadatas"."id";--> statement-breakpoint
+
+--- Remove evaluation metadatas with NULL configuratios (these are llm_as_judge_evaluation_metadatas rows that were left when removing the original evaluation)
+DELETE FROM "latitude"."llm_as_judge_evaluation_metadatas" WHERE "latitude"."llm_as_judge_evaluation_metadatas"."configuration" IS NULL--> statement-breakpoint
+
+--- Set configuration column as NOT NULL
+ALTER TABLE "latitude"."llm_as_judge_evaluation_metadatas" ALTER COLUMN "configuration" SET NOT NULL;
```

Comment:
Moving configuration from `evaluations` to `llm_as_judge_evaluation_metadatas`, and then removing the `llm_as_judge_evaluation_metadatas` that were left over after removing their original `evaluations`